server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java

```diff
@@ -30,4 +32,21 @@ public interface SessionOperations {
 
     void killQuery(@NonNull Connection connection, @NonNull String connectionId);
 
+    /**
+     * Get kill query SQL by connectionId
+     *
+     * @param connectionIds
+     * @return the map of connectionId to kill query SQL
+     */
+    Map<String, String> getKillQuerySqls(@NonNull List<String> connectionIds);
```

Comment:
the return value `Map<String, String>` is not easy to understand, how about this:
```Java
String getKillQuerySql(@NonNull String connectionId);
String getKillSessionSql(@NonNull String connectionId);
```

Reply:
Then the calling side needs to handle the scenario of getting SQLs in batch, which is not easy to use.
I think as the input parameter is a list, the return value, a map, does not seem that hard to understand.

Reply:
SessionOperations.java is used in SessionExtensionPoint which is our plugin interface and it is not a internal interface~, this interface may be impled by out developer. We should not design interfaces only from the perspective of our own use.

Reply:
I will change it by this way: 
I create two single methods: `getKillQuerySql` and `getKillSessionSql`. The batch method will have a default implementation in the interface which relies on these two single methods. And the batch methods' input has been changed to a `Set`.